hotsos/core/plugins/system/system.py

```diff
@@ -128,6 +129,77 @@ def unattended_upgrades_enabled(self):
 
         return False
 
+    @cached_property
+    def ubuntu_pro_status(self):
+        """Parse and retrieve Ubuntu Pro status
+
+        Returns:
+            Dictionary: Ubuntu pro status as a dictionary, e.g.::
+            {
+                "services": {
+                    "esm-apps": {
+                        "entitled": "yes",
+                        "status": "enabled"
+                    }, /* ... */
+                },
+                "account": "ACME Corporation",
+                "subscription": "Ubuntu Pro (Apps-only) - Virtual",
+                "technical_support_level": "essential",
+                "valid_until": "Sat Jan  1 01:01:01 9999 +03"
+            }
+
+            None: `pro/ua` command not available
```

Comment:
this is not quite true since if it is not enabled i currently get ```ubuntu-pro: false```

Reply:
Yep, I assumed the cli helper would return "None" when the command is not present, but that's not the case. Removed it in the last revision.